applications/GeoMechanicsApplication/custom_elements/steady_state_Pw_interface_element.hpp

```diff
@@ -87,8 +86,7 @@ class KRATOS_API(GEO_MECHANICS_APPLICATION) SteadyStatePwInterfaceElement
     {
     }
 
-    /// Destructor
-    ~SteadyStatePwInterfaceElement() override {}
+    ~SteadyStatePwInterfaceElement() override = default;
```

Comment:
I don't think there is a need to use the `override` keyword in combination with destructors.

Reply:
Done